### ISSUE TITLE
Provide initial hint to understand the environment

### ISSUE THREAD

**Issue by ajay-sh21** [w3c/apa#72]
While accessing the virtual environment with screen readers, , I think there should be a bit overview about the environment first so that users could expect what kind of environment is it, and hence act based on this information. This 	information could be something similar to , a one liner description that could be given to  a blind person who just entered the room e.g. “A conference room with some people sitting around a long table that has some empty chairs around it”.

**Comment by RealJoshue108** [comment]
Thanks @ajay-sh21 This kind of 'orientation' information could be useful to a range of users.

**Comment by RealJoshue108** [comment]
We discussed in APA - there is a need for description by another name - or providing an overview before engaging. There is an issue around there being so many objects in XR - that they need to be filtered. If object types are varied, it is good to have an overview but it may also be too much. 

API mechanisms can be used, and pushed to the top so the user 'gets that' straight away. For this kind of use case, a similar use case is a movie that has an extended text description, via a TTS setting, where more time can be granted to the user to consume a description before they do into the XR environment. This was dealt with in the Media Accessibility User Requirements (MAUR).

**Comment by RealJoshue108** [comment]
https://www.w3.org/TR/media-accessibility-reqs/

**Comment by RealJoshue108** [comment]
So this is a similar use case - this kind of pattern would indeed be helpful in XR, in education and other contexts. So the idea is that you can stall time, to consume some alternative media format. @ajay-sh21 

We will actually look at working this kind of thing into a new version of XAUR.

**Comment by ajay-sh21** [comment]
Sure, this would be great!

**Comment by RealJoshue108** [comment]
Hi @ajay-sh21 some of your input has resulted in a new draft requirement in the updated XAUR. Can you please let me know your name and affiliation, so I can add you to the acknowledgements? You can do that here - or ping me on joconnor@w3.org

**Comment by ajay-sh21** [comment]
Thanks @RealJoshue108 for considering this. I'll ping you the details.
